Src/FluentAssertions/Extensions/FluentTimeSpanExtensions.cs

```diff
@@ -47,6 +47,14 @@ public static TimeSpan Ticks(this long ticks)
         return TimeSpan.FromTicks(ticks);
     }
 
+    /// <summary>
+    /// Returns a <see cref="TimeSpan" /> with an added amount of ticks.
+    /// </summary>
+    public static TimeSpan AddTicks(this TimeSpan timeSpan, int ticks)
+    {
+        return TimeSpan.FromTicks(timeSpan.Ticks + ticks);
+    }
```

Comment:
Let's keep this an internal method for now to keep the PR focused on a single feature.

Consumers can currently use `timeSpan + 1.Ticks()` or `timeSpan.Add(1.Ticks())`.

Reply:
Ok